oar/core/worksheet.py

```diff
@@ -146,21 +141,48 @@ class TestReport:
     def __init__(self, ws: Worksheet, cs: ConfigStore):
         self._ws = ws
         self._cs = cs
+        self.wm = WorksheetManager(cs)
 
     def get_url(self):
         """
         Get worksheet url of the report
         """
         return self._ws.url
 
-    def update_advisory_info(self, ad):
+    def update_advisory_info(self):
         """
         Update advisory info in test report
-
-        Args:
-            ad (str): advisories of current release
         """
-        self._ws.update_acell(LABEL_ADVISORY, ad)
+        obj = []
+        # Arrange AD names and urls as per needs of spreadsheet.batch_update API
+        for k, v in self._cs.get_advisories().items():
+            obj.append({"ad_name": k, "ad_number": str(v), "ad_url": util.get_advisory_link(v) })
+
+        # Extract only the AD names and number
+        text = "\n".join([e["ad_name"] + ": " + e["ad_number"] for e in obj])
+        spreadsheet =  self.wm._doc
+        sheet = spreadsheet.worksheet(self.wm._cs.release)
+        requests = [
+            {
+                "updateCells": {
+                    "rows": [
+                        {
+                            "values": [
+                                {
+                                    "userEnteredValue": {"stringValue": text},
+                                    "textFormatRuns": [{"format": {"link": {"uri": e["ad_url"]}}, "startIndex": text.find(e["ad_name"])} for e in obj],
+                                }
+                            ]
+                        }
+                    ],
+                    "range": {"sheetId": sheet.id, "startRowIndex": 1, "endRowIndex": 2, "startColumnIndex": 1, "endColumnIndex": 2},
+                    "fields": "userEnteredValue,textFormatRuns,userEnteredFormat"
```

Comment:
```suggestion
                    "fields": "userEnteredValue,textFormatRuns"
```
`userEnteredFormat` is redundant here.